app/views/docs/functions.phtml

```diff
@@ -384,6 +384,35 @@ func main(req: RequestValue, res: RequestResponse) throws -> RequestResponse {
     <p>If you're running Appwrite locally and trying to connect to your local Appwrite instance from a cloud function, you'll have to use your local IP address in place of localhost. This is because, within the container localhost does not refer to the host machine rather the container itself. </p>
 </div>
 
+<h2><a href="/docs/functions#ignoreFiles" id="ignoreFiles">Ignore Files</a></h2>
+
+<p>Libraries folder such as <b>node_modules</b> or <b>vendor</b> should not be included in the tarball, although you want them locally to have full IDE support. Similarly, you should keep files holding secrets out of your deployment. You can use the Appwite CLI feature <b>file ignoring</b> to exclude specific files from a deployment.</p>
```

Comment:
I would avoid calling our packaged function a `tarball`. What term have we used in other places?